org.icculus.tennix.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: org.icculus.tennix
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: tennix
+rename-icon: tennix
+rename-desktop-file: tennix.desktop
+rename-appdata-file: tennix.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: sdl12-compat
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz
+        sha256: 5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014
+
+  - shared-modules/SDL/SDL_image-1.2.12.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
+  - shared-modules/SDL/SDL_net-1.2.8.json
+  - shared-modules/SDL/SDL_ttf-2.0.11.json
+
+  - name: tennix
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix ${FLATPAK_DEST}
+      - make install
+      - install -Dm0644 data/tennix.desktop ${FLATPAK_DEST}/share/applications/tennix.desktop
+      - install -Dm0644 tennix.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/tennix.svg
+      - install -Dm0644 tennix.appdata.xml ${FLATPAK_DEST}/share/metainfo/tennix.appdata.xml
```

Comment:
I will reiterate what I said previously in previous reviews: you should have it properly named, with the proper app-id.

`tennix.desktop` is NO LONGER a valid app-id.

Reply:
This is a very old piece of software. It is way more practical to use the Flatpak rename scripts than sedding all the files and its mentionings manually.

Reply:
You literally need to replace `tennix` with ID or `$FLATPAK_ID` in these 3 lines.

Reply:
My comment is just for the appdata. `rename-icon` will rename in the .desktop, and `rename-desktop-file` will rename the launchable in the appdata, so there is a benefit for either.
However the appdata has no dependency so it should be written with the proper app-id and have the proper name.